bigbluebutton-html5/imports/ui/components/nav-bar/settings-dropdown/component.jsx

```diff
@@ -97,8 +97,8 @@ class SettingsDropdown extends Component {
     let fullScreenIcon = 'fullscreen';
 
     if (isFullScreen) {
-      fullScreenLabel = intl.formatMessage(intlMessages.exitFullScreenLabel);
-      fullScreenDesc = intl.formatMessage(intlMessages.exitFullScreenDesc);
+      fullScreenLabel = intl.formatMessage(intlMessages.exitFullscreenLabel);
```

Comment:
Shouldn't is be exitFullScreenLabel ?
It looks strange - on the left fullScreenLabel is camel case, and on the right exitFullscreen is not.